src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import './App.css'
+import React from 'react'
+import { HomePage } from './pages/HomePage/HomePage'
+
+export function App() {
+  return (
+    <>
+    <div>
+      <HomePage />
+    </div> 
+    </>
```

Comment:
Acredito que neste caso não é necessário o uso do fragment nem da div. Pode deixar apenas o componente HomePage 😊
```suggestion
    <HomePage />
```